Source/Scene/Cesium3DTile.js

```diff
@@ -733,6 +733,8 @@ define([
         this.replacementNode = undefined;
 
         this.lastStyleTime = 0;
+        this.clippingPlanesDirty = this._clippingPlanesState === 0;
```

Comment:
We don't chain values like this, but since you're already setting `this._clippingPlanesState` on the next line, I assume this is a copy/paste error.

Reply:
This isn't chaining values.  It's setting `clippingPlanesDirty` to a boolean if `_clippingPlanesState` is equal to 0

Reply:
Ahh, I missed that.  Still this line is confusing, perhaps put the right hand side in parenthesis to make it more obvious.